samples/c_cxx/MNIST/MNIST.cpp

```diff
@@ -0,0 +1,228 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#define UNICODE
```

Comment:
and define _UNICODE ?

Reply:
Windows uses #ifdef UNICODE, does it also use _UNICODE?